WordPress/Classes/ViewRelated/Stats/Shared Views/Post Stats/PostStatsTableViewController.swift

```diff
@@ -52,7 +52,8 @@ class PostStatsTableViewController: UITableViewController, StoryboardLoadable {
             return nil
         }
 
-        cell.configure(date: selectedDate, period: .day, delegate: self)
+        let periodCount = SiteStatsTableHeaderView.defaultPeriodCount
+        cell.configure(date: selectedDate, period: .day, delegate: self, expectedPeriodCount: periodCount)
```

Comment:
Just a thought - if you give `expectedPeriodCount` in `configure`a default value, like maybe `expectedPeriodCount: Int = defaultPeriodCount`, this won't be necessary since it's just setting the count to the default.

Reply:
Thanks for the suggestion @ScoutHarris! I have pushed a commit that takes your suggestion into account.